website/content/en/preview/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -79,6 +79,11 @@ The following cluster configuration will:
 
 {{% script file="./content/en/{VERSION}/getting-started/getting-started-with-karpenter/scripts/step06-add-spot-role.sh" language="bash"%}}
 
+{{% alert title="Note" color="primary" %}}
+In order to run Windows workloads, Windows support should be enabled in your EKS Cluster.
+See documentation [here](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for how to enable Windows support.
```

Comment:
I'd write:
```suggestion
See [Enabling Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) to learn more.
````

(that's roughly following a version of the Kubernetes style guide BTW)